Automattic/jetpack
# Coming Soon: Change the color on the coming soon banner to new WordPress blue

Fixes https://github.com/Automattic/dotcom-forge/issues/9468

## Proposed changes:

This PR changes the color of the Coming Soon banner to the new WordPress blue which is `#3858E9`.
### Other information:

- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them? N/A

## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Connect to your sandbox
* Create a Simple WP.com site with a Premium plan and do not launch it (to ensure it is still in the coming soon mode)
* On your sandbox, run `cd public_html`
* Next, run the command from below:

```
bin/jetpack-downloader test jetpack-mu-wpcom-plugin fix/color-on-wp-coming-soon-banner
```
* Sandbox the site that you have just created (no need to sandbox public API)
* Navigate to the site's address in the incognito window
* Confirm that you see the new WordPress Blue color on the coming soon banner:

<img width="1881" alt="Screenshot 2024-10-30 at 3 54 25 PM" src="https://github.com/user-attachments/assets/66bf5b20-7199-4386-8dd6-0508aaee6c66">

* I did not add the testing instructions for WoA sites because it is the same banner that is loaded there as this one. Happy to add those if considered needed
